server/index.js

```diff
@@ -16,13 +16,25 @@ const app = express();
 const User = require("./models/user-schema");
 var morgan = require("morgan");
 
+const user = require("./models/user-schema");
```

Comment:
This is a little bit redundant since User was already defined prior. Make sure to use the same constants (i.e. User instead of user) to keep everything structured.